src/lib/draw/StreetViewMode.svelte

```diff
@@ -15,38 +9,18 @@
 
   const thisMode = "street-view";
 
-  const circleRadiusPixels = 10;
-
   export let changeMode: (m: Mode) => void;
 
-  export function start() {}
-  export function stop() {
-    cursor = null;
+  export function start() {
+    $map.getCanvas().style.cursor = "zoom-in";
```

Comment:
Maybe this is a strange icon to use for street view? Open to suggestions. Built-in options at https://developer.mozilla.org/en-US/docs/Web/CSS/cursor, and we can use any custom icon.

Reply:
I can't see a better one listed